### ISSUE TITLE
Bundle the JSon libraries with the plug-in.

### ISSUE THREAD

**Issue by thegreystone** [thegreystone/jmc-flame-view#7]
It would be nice to be able to work with the Flame Graph view, even when there is no direct connection to https://cdnjs.cloudflare.com. That said, it may be an issue getting at the resources as they are in a jar, and need to be referenced from the HTML. One solution would be to just write the JavaScript libraries to a well known location (a well known folder in temp for example). That however may be considered a security risk.

This bug is about figuring out how to best accomplish bundling the JSon libraries with the plug-in. ;)

**Comment by nbogojevic** [comment]
One way to work around this is to launch local server that listens on random ephemeral port and serves all this content from class path resources. 
All javascript libraries would be bundled inside (maybe along page.html file). Browser in view would then target the server url instead of using setText.

**Comment by thegreystone** [comment]
Yep, that is one alternative. It would be interesting to see if such a solution would have different performance characteristics.

**Comment by nbogojevic** [comment]
Did some test around this approach. Subjective feeling is that actually seems faster to render when content is served via server, there are also some advantages as javascript code can be well cached by browser, but I have to test it little bit more. 

The approach also allows to work around #4 - in that one I think the issue is that eclipse uses internet explorer to render web pages on all versions of Windows. By using embeded server, I can open it in external browser (Chrome, Firefox, Edge) and it renders correctly.

**Comment by thegreystone** [comment]
Oh gods. There should be a bug open on Eclipse for embedding Edge instead of Internet Explorer when available. ;)

**Comment by thegreystone** [comment]
I opened one: 
https://bugs.eclipse.org/bugs/show_bug.cgi?id=538991
